astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -284,8 +322,8 @@ def test_lstrip_whitespace(parallel, read_basic):
     text = """
      1,  2,   \t3
  A,\t\t B,  C
-  a, b,   c
-""" + '  \n'
+  a, b,   c  \n
+"""
```

Comment:
```suggestion
  a, b,   c
  
"""
```
Got the newline in the wrong place here. Obviously it makes no difference to the test, but since the original produced a whitespace line at the end, it's probably best to reproduce that.